apps/webview/src/components/Global/Monitoring.tsx

```
@@ -0,0 +1,55 @@
+      <GoogleAnalytics gaId={config.monitoring.GA} />
+      <GoogleTagManager gtmId={config.monitoring.GTM} />
+      <Analytics />
+
+      <Script
+        strategy="afterInteractive"
+        src={`https://d-collect.jennifersoft.com/${MONITORING_KEY.JENNIFER}/demian.js`}
+      />
+      <Script
+        id="jennifer-init"
+        strategy="afterInteractive"
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)